gffutils/test/test.py

```diff
@@ -1230,6 +1230,8 @@ def test_issue_119():
     assert db5._autoincrements == {'gene': 2}
     assert db3._autoincrements == db5._autoincrements
 
+    assert db3._autoincrements == db5._autoincrements
```

Comment:
This is a duplicate of the above line ;)

Reply:
Fixed!